JSDemos/Demos/DataGrid/CellEditingAndEditingAPI/Angular/app/app.component.ts

```diff
@@ -33,12 +34,33 @@ export class AppComponent {
 
     selectionChanged(data: any) {
         this.selectedItemKeys = data.selectedRowKeys;
+        this.deleteButton.option("disabled", !data.selectedRowsData.length);
     }
+
     deleteRecords() {
         this.selectedItemKeys.forEach((key) => {
             this.dataSource.remove(key);
         });
         this.dataGrid.instance.refresh();
+        this.deleteButton.option("disabled", true);
+    }
+
+    onToolbarPreparing(e) {
+        e.toolbarOptions.items[0].showText = 'always';
+
+        e.toolbarOptions.items.push({
+            location: "after",
+            widget: "dxButton",
+            options: {
+                text: "Delete Selected Records",
+                icon: "trash",
+                disabled: true,
```

Comment:
Try to use toolbar item with template instead of widget dxButton. In this case it is possible to use bindings for disabled property to avoid using option method.